win32ss/user/user32/windows/text.c

```diff
@@ -148,13 +148,25 @@ LPSTR
 WINAPI
 CharPrevA(LPCSTR start, LPCSTR ptr)
 {
-    while (*start && (start < ptr))
+    if (ptr > start)
```

Comment:
What if `ptr < start`, should it still return `ptr` then?

Reply:
Yes, in this case it has to return just `ptr`, proven by API tests.